docgen/content-sources/node/toc.yaml

```diff
@@ -115,25 +117,13 @@ toc:
   path: /docs/reference/admin/node/admin.credential
   section:
   - title: "Credential"
-    path: /docs/reference/admin/node/admin.credential.Credential
+    path: /docs/reference/admin/node/admin.credential.Credential-1
```

Comment:
Sorry if I missed something obvious! Why do we suffix these with `-1`?

Reply:
That's the name of the auto-generated file now.

Reply:
Is that something we want, or is it kind of forced on us?

Reply:
Not by choice. It was the name auto picked by typedoc

Reply:
I believe this is because we have merged declarations. `credential` is both a module and a namespace. TypeDoc is probably not handling that well.